.rubocop.yml

```diff
@@ -1,25 +1,3 @@
-require: rubocop-rspec
```

Comment:
the rules in `.rubocop.yml` are not used... you are now enabling for example `RedundantReturn` which we agreed to disable